plugins/content_distribution/providers/facebook/lib/FacebookDistributionEngine.php

```diff
@@ -72,18 +76,24 @@ protected function doSubmit(KalturaDistributionSubmitJobData $data)
 				filesize($videoPath),
 				$this->tempDirectory,
 				$facebookMetadata);
-		} catch (Exception $e) {
+		}
+		catch (Exception $e)
+		{
 			throw new Exception("Failed to submit facebook video , reason:".$e->getMessage());
 		}
 
-		if ($data->providerData->captionsInfo)
+		$tags = FacebookDistributionEngine::helperGetTags($data->providerData->fieldValues);
```

Comment:
Accessing static function in the same class should be made with self:: operator.